src/EnergyPlus/SolarShading.cc

```diff
@@ -7969,7 +7986,7 @@ namespace SolarShading {
 
 		for ( ISurf = 1; ISurf <= TotSurfaces; ++ISurf ) {
 			SurfaceWindow( ISurf ).ExtIntShadePrevTS = SurfaceWindow( ISurf ).ShadingFlag;
-			SurfaceWindow( ISurf ).ShadingFlag = NoShade;
+			// SurfaceWindow( ISurf ).ShadingFlag = NoShade;
 			SurfaceWindow( ISurf ).FracTimeShadingDeviceOn = 0.0;
```

Comment:
Delete?  If there's a good reason to keep this for clarity, then please add a comment to explain.